packages/backend-proxy-middleware/package.json

```diff
@@ -0,0 +1,58 @@
+{
+    "name": "@sap-ux/backend-proxy-middleware",
+    "description": "SAP backend proxy middleware",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/backend-proxy-middleware"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Abackend-proxy-middleware"
+    },
+    "version": "0.4.0",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "clean": "rimraf dist",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "watch": "tsc --watch"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "ui5.yaml"
+    ],
+    "dependencies": {
+        "@sap-ux/axios-extension": "workspace:*",
+        "@sap-ux/btp-utils": "workspace:*",
+        "@sap-ux/logger": "workspace:*",
+        "@sap-ux/store": "workspace:*",
+        "@sap-ux/ui5-config": "workspace:*",
```

Comment:
We are not using anything from this package. Please remove

Reply:
Done